product/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from .views import ProductListView, ProductDetailView
+
+urlpatterns = [
+        path("/goods_list", ProductListView.as_view()),
```

Comment:
goods 라고만 해도 될것같습니다. 그리고 list 써도 될것같은데 쓰더라도 언더스코어보다는 엔드포인트 주소는 - 로 구분해주세요.
```suggestion
        path("/goods", ProductListView.as_view()),
```